### PR TITLE
Update enable-usage-analytics.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the navigation to initiate the template  for usage analytics

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9999

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/admin/usage-analytics/enable-usage-analytics.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ To enable Microsoft 365 usage analytics in a Microsoft 365 US Government Communi
 
 ## Before you begin
 
-To get started with Microsoft 365 usage analytics you must first make the data available in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">Microsoft 365 admin center</a>, then initiate the template app in Power BI.
+To get started with Microsoft 365 usage analytics you must first make the data available in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">Microsoft 365 admin center</a>, Select **Reports** -> **Usage** then initiate the template app in Power BI.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
To get started with Microsoft 365 usage analytics you must first make the data available in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">Microsoft 365 admin center</a>, then select **Reports** > **Usage** and initiate the template app in Power BI.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this Pr. Thanks

**Pull request merged**